test/3x/common/test_logger.py

```diff
@@ -29,6 +51,18 @@ def test_logger(self):
         logger.warning(({"msg": "call logger warning function"}, {"msg2": "done"}))
         logger.warning(({"msg": [{"sub_msg": "call logger"}, {"sub_msg2": "call warning function"}]}, {"msg2": "done"}))
 
+    def test_logger_func_and_pretty_dict(self):
+        from neural_compressor.common.utils import debug, error, fatal, info, log, warning
```

Comment:
Which format is more recommended?  `from neural_compressor.common.utils import debug` or `from neural_compressor.common import logger`?

Reply:
The purpose of `from neural_compressor.common import debug` is to make the logger dict pretty.
Here is a demo:
```python
FP32 = {"weight": {"dtype": ["fp32"]}, "activation": {"dtype": ["fp32"]}}

from neural_compressor.common import logger
logger.info(FP32)
# 2024-01-31 09:21:19 [INFO][log_demo.py:4] {'weight': {'dtype': ['fp32']}, 'activation': {'dtype': ['fp32']}}


logger.info("---------------------")
from neural_compressor.common import info
info(FP32)
# 2024-01-31 09:21:19 [INFO][log_demo.py:8] {
# 2024-01-31 09:21:19 [INFO][log_demo.py:8]     'weight': {
# 2024-01-31 09:21:19 [INFO][log_demo.py:8]         'dtype': [
# 2024-01-31 09:21:19 [INFO][log_demo.py:8]             'fp32'
# 2024-01-31 09:21:19 [INFO][log_demo.py:8]         ]
# 2024-01-31 09:21:19 [INFO][log_demo.py:8]     },
# 2024-01-31 09:21:19 [INFO][log_demo.py:8]     'activation': {
# 2024-01-31 09:21:19 [INFO][log_demo.py:8]         'dtype': [
# 2024-01-31 09:21:19 [INFO][log_demo.py:8]             'fp32'
# 2024-01-31 09:21:19 [INFO][log_demo.py:8]         ]
# 2024-01-31 09:21:19 [INFO][log_demo.py:8]     }
# 2024-01-31 09:21:19 [INFO][log_demo.py:8] }
```

Reply:
Perhaps we should move the functionality of the pretty dict to logger and only maintain one usage: `from neural_compressor.common import logger`.

Reply:
Only keep `from neural_compressor.common import logger`.